flowzone.yml

```diff
@@ -2787,13 +2787,31 @@ jobs:
           echo "CF_BRANCH=${{ github.event.repository.default_branch }}" >> "${GITHUB_ENV}"
 
       - name: Deploy to Cloudflare Pages
+        id: deploy_cf_pages
         uses: cloudflare/wrangler-action@7c01cc48d88a1f83b4716e0557c168854a9dcdb8 # v3.4.1
         with:
           apiToken: ${{ secrets.CF_API_TOKEN }}
           accountId: ${{ secrets.CF_ACCOUNT_ID }}
           wranglerVersion: "3.5.1" # latest - https://www.npmjs.com/package/wrangler
           command: --version
           postCommands: npx wrangler pages deploy --branch ${{ env.CF_BRANCH }} --project-name=${{ inputs.cloudflare_website }} build/ | tee -a $GITHUB_STEP_SUMMARY
+          
+      - name: Find Comment
+        uses: peter-evans/find-comment@v3
+        id: find_cf_comment
+        with:
```

Comment:
@drskullster this will probably fail because the default GH token doesn't have PR write permissions, for inspiration on how to fix, please check [this](https://github.com/product-os/flowzone/pull/957/files).